src/containers/Token/TokenHeader/test/actions.test.js

```diff
@@ -1,108 +0,0 @@
-import configureMockStore from 'redux-mock-store'
-import thunk from 'redux-thunk'
-import * as actions from '../actions'
-import * as actionTypes from '../actionTypes'
-import { initialState } from '../reducer'
-import { NOT_FOUND, BAD_REQUEST, SERVER_ERROR } from '../../../shared/utils'
-import rippledResponses from './rippledResponses.json'
-import actNotFound from './actNotFound.json'
-import MockWsClient from '../../../test/mockWsClient'
-
-const TEST_ADDRESS = 'rDsbeomae4FXwgQTJp9Rs64Qg9vDiTCdBv'
-const TEST_CURRENCY = 'abc'
-
-describe('TokenHeader Actions', () => {
-  jest.setTimeout(10000)
-
-  const middlewares = [thunk]
-  const mockStore = configureMockStore(middlewares)
-  let client
-  beforeEach(() => {
-    client = new MockWsClient()
-  })
-
-  afterEach(() => {
-    client.close()
-  })
-
-  it('should dispatch correct actions on successful loadTokenState', () => {
-    client.addResponses(rippledResponses)
```

Comment:
it might be worth having these tests still to check that rippled responses are transformed right.

Reply:
We can have a ticket to add these back later.